app/addons/activetasks/stores.js

```diff
@@ -0,0 +1,127 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  "app/helpers",
+  "api",
+  "addons/activetasks/actiontypes"
+], function (Helpers, FauxtonAPI, ActionTypes) {
+
+  var ActiveTasksStore = FauxtonAPI.Store.extend({  
+    //elements in here
+
+    getSelectedTab: function () {
+      return this._selectedTab;
+    },
+    setSelectedTab: function (selectedTab) {
+      this._selectedTab = selectedTab;
+    },
+    getPollingInterval: function () {
+      return this._pollingInterval;
+    },
+    setPollingInterval: function (pollingInterval) {
+      this._pollingInterval = pollingInterval;
+    },
```

Comment:
Pardon my curiosity, but all these `getFoo` / `setFoo` shouldn't be defined via [Object.defineProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty) or [\_\_defineGetter\_\_ / \_\_defineSetter\_\_](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__) instead for more nicer, java-less, usage?